model/task.js

```diff
@@ -18,9 +18,10 @@ const MODEL = {
         .description('Container this Task is running in')
         .example('node:4'),
 
-    environment: Joi
-        .object()
-        .description('Key=>Value environment variables for the task itself'),
+    environmentId: Joi
```

Comment:
Just confirming: does your other PR handle the order of env? For example: we expect 
`{a:1; b:2}` and `{b:2; a:1}` to have the same environmentId.